src/index.js

```diff
@@ -1,3 +1,6 @@
+import "react-toastify/dist/ReactToastify.min.css";
```

Comment:
does this really need to be here?

Reply:
as in? Put it elsewhere?

Reply:
yes maybe we can have it in the file `src/Message/Message.js`? i have the feeling it is more part of the component than part of the exports.